indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -114,6 +114,14 @@
                                                 {% trans %}No track{% endtrans %}
                                             {% endif %}
                                         </td>
+                                    {% elif item.id == 'score' %}
+                                        <td class="i-table" data-text="{{ abstract.score }}">
+                                            {% if abstract.accepted_track %}
```

Comment:
Why check for an accepted track? You can have a score without having a final judgment.